Update ESPUI.cpp

sunlord2k opened the pull request:
Changed FILE_WRITE to FILE_WRITING for compatibility with SD.h for usage with SD cards

MartinMueller2003 reviewed (commented):
FILE_WRITE is an Alias for "w" and is defined locally in ESPUI.h 
Are you saying the definition conflicts with SD.h? If so, changing the usage does not resolve the conflict. You still have the FILE_WRITE definition in place. Where is FILE_WRITING defined? What is its value?

sunlord2k commented:
Hi Martin,

I put in two pull requests. FILE_WRITE is also defined in the SD.h leading to conflicts if you use ESPUI and SD.h for SD card usage in one project. That's why I changed the definitions name to FILE_WRITING and checked the ESPUI code for every occurence of FILE_WRITE  and renamed it as well (see second pull request).

I tested it and it works fine now.

Am 5. April 2024 13:51:10 MESZ schrieb Martin Mueller ***@***.***>:
>@MartinMueller2003 commented on this pull request.
>
>FILE_WRITE is an Alias for "w" and is defined locally in ESPUI.h 
>Are you saying the definition conflicts with SD.h? If so, changing the usage does not resolve the conflict. You still have the FILE_WRITE definition in place. Where is FILE_WRITING defined? What is its value?
>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/s00500/ESPUI/pull/299#pullrequestreview-1982958867
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>

MartinMueller2003 reviewed (approved):
This is fine.
